PWGJE/EMCALJetTasks/CMakeLists.txt

```diff
@@ -319,6 +319,7 @@ if(FASTJET_FOUND)
         UserTasks/AliAnalysisTaskEmcalSubjet.cxx
 	    UserTasks/AliAnalysisTaskJetCoreEmcal.cxx
         UserTasks/AliAnalysisTaskEmcalJetCDF.cxx
+	UserTasks/AliAnalysisTaskEmcalJetProperties.cxx
```

Comment:
Please make the indentation uniform with the rest of the lines

Reply:
Here also I followed this task: AliAnalysisTaskEA.cxx
Do I need to add something?
Kindly help me.

Reply:
I would like to inform you that from my browser it seems fine. I am attaching a screenshot.
<img width="1440" alt="Screenshot 2022-07-11 at 5 05 34 PM" src="https://user-images.githubusercontent.com/107457495/178255753-0206de01-bf8b-43d6-a114-d034dd9537bb.png">

Reply:
The line you added is indented less than the surrounding lines. Just make it the same level of indentation. It's just formatting:

```
        UserTasks/AliAnalysisTaskEmcalJetCDF.cxx
        UserTasks/AliAnalysisTaskEmcalJetProperties.cxx
```

I guess you probably used a literal tab, so please replace the tab with spaces (which I did in the example above)

Reply:
Weird, github has a tabstop bug:

<img width="896" alt="Screen Shot 2022-07-11 at 3 59 51 PM" src="https://user-images.githubusercontent.com/1571927/178281620-d7b5b47c-2ffa-4494-8763-99e3882fbecf.png">

You could fix it by the way that I described (tab -> spaces), but okay, it's fine

Reply:
I tried to fix it. While changing in my code it seems okay, but after comitting it changes again!